docs/metrics.md

```diff
@@ -425,15 +400,15 @@ Add time-based by invoking the following method:
 Function API
 
 ```python
-    def add_time(self, name: str, value: int or float, idx=None, unit: str = 'ms', dimensions: list = None,
+    def add_time(self, metric_name: str, value: int or float, idx=None, unit: str = 'ms', dimensions: list = None,
                  metric_type: MetricTypes = MetricTypes.GAUGE):
         """
         Add a time based metric like latency, default unit is 'ms'
             Default metric type is gauge
 
         Parameters
         ----------
-        name : str
+        metric_name : str
```

Comment:
Same as above

Reply:
It looks like for these changes, I might have been using a slightly older metrics.md before you made your changes. I have updated this.